Bottom: cc12a1b7d1fab0beae73df470afa0676b0f58723
Top:    ef180055e9732d8db5b6a19e0577324a7b3c85b3
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 14:35:16 +0800

Refresh of autofs-5.0.7-fix-master-map-mount-options-matching.patch

---

diff --git a/lib/master_tok.l b/lib/master_tok.l
index 4c0cc1a..8d1f1a2 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -91,7 +91,7 @@ OPTWS		[[:blank:]]*
 NL		\r?\n
 CONT		\\\n{OPTWS}
 
-OPTIONSTR      ([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=(\"?([[:alnum:]_\-\:])+\"?))?)+)
+OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=(\"?([[:alnum:]_\-\:])+\"?))?)+)
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
